examples/encfs/README.md

```diff
@@ -55,7 +55,7 @@ Replace [managed-identity-with-right-permissions-to-key-vault](aci-arm-template.
 
 #### 3. Populate Image Registry Credentials
 
-Update the [image registry credentials](aci-arm-template.json?plain=1#L87) on the ARM template in order to access a private container registry. The credential could be either a managed identity or username/password. This section is not needed for public images. 
+Update the [image registry credentials](aci-arm-template.json?plain=1#L94) on the ARM template in order to access a private container registry. The credential could be either a managed identity or username/password. This section is not needed for public images. If the credential needs to be a managed identity, the managed identity needs to be granted access to the private images. 

```

Comment:
I would show what this looks like in the arm template here

Reply:
Yeah the arm template is also updated.

Reply:
No, what I mean is to show how you can use the managed identity OR the username/password here, since you removed username/password from the arm template.